release

```diff
@@ -50,6 +50,15 @@ run() {
   git push origin production
 
   git checkout main
+
+  local recent_tags=$(git for-each-ref --sort=-creatordate --format '%(refname)' refs/tags | head -n 2)
+  local previous_tag=$(echo "$recent_tags" | tail -n 1)
+  local this_tag=$(echo "$recent_tags" | head -n 1)
+
+  if [[ "$previous_tag" != '' && $this_tag != '' ]]; then
+    echo "ℹ️ Changes in this release (please share a changelog with the team):"
+    git log $previous_tag..$this_tag^
```

Comment:
Sorry for the drive-by review, but I wonder if `--oneline` would lead to more readable output since it wouldn't include the author, etc?

Reply:
@scotttrinh I thought about it but then you lose the commit body... then I was briefly playing with custom formatting but got a bit 🐰 🕳️ ...

It's interesting to see everyone's different approaches to PR merge messages; it looks like yours usually don't include a body (in which case online would definitely make more sense!) whereas I usually copy the PR description into the squashed commit (probably why I leaned towards this) - everyone else is somewhere in the middle.

so... i'm not thoroughly convinced one way or the other is "better"  but I wonder if having to do the changelog might encourage us to make those commit bodies more descriptive in the first place? 🤷

Reply:
Yeah, that's interesting. I guess my only feeling is that whatever this outputs, is exactly what we should copy into whatever place it needs to be copied into without any editing, so this seems fine.

As far as whether commit message bodies are something that we should care about, I'd love to have more discussion here! I find that so much of the context of the change happens in the communication in the PR itself, so it's hard to rely on the PR description alone to get the gist of the change, but knowing that they're externally facing might help guide how we write these descriptions.

Reply:
I've thought about it a bit more and I'm cool with a short one

`--oneline` also includes less useful stuff like the hash, and doesn't include author/date, so i dipped my toe in the 🐰 🕳️  a tiny bit and came up with 1e6b94d — let me know what you think

sample output:
```
- 2021-08-17 (Dylan Pyle) Shorter log
- 2021-08-17 (Dylan Pyle) Add diff log to `release`
- 2020-07-01 (Dylan Pyle) Fix package.json
```